nixos/doc/manual/release-notes/rl-1803.xml

```diff
@@ -95,6 +95,15 @@ following incompatible changes:</para>
       <link xlink:href="https://search.nix.gsc.io/?q=stateVersion">here</link>.
     </para>
   </listitem>
+  <listitem>
+    <warning>This belongs in Nixpkgs release notes, were we to have it</warning>
```

Comment:
TODO to me to create this, or just remove the warning

Reply:
Motivation: For another different one, I want to hyperlink a nixpkgs manual section, which is only readily done from within the nixpkgs manual.

Reply:
I think we've intentionally restricted nixpkgs/NixOS release notes to NixOS as there is hardly any code which could not be added to both and we don't technically have releases for nixpkgs,

Reply:
The same argument would seem to work for merging the two manuals :-)

Reply:
I think I'd agree, simply by looking at my behaviour of grepping through both every time I look for something. But that discussion should not start here, but rather on the mailing list or an RFC. :)